apps/i18n/music/en_us.json

```diff
@@ -58,6 +58,7 @@
   "chordArpeggioRandom": "Arpeggio Random",
   "chordTogether": "Together",
   "feedback": "Feedback",
+  "madeWithMusicLabOn": "made with Music Lab on",
```

Comment:
Per a [discussion](https://docs.google.com/document/d/10YgMuJCYqjCGqrmBdxESVnARJ6G_TPTeGv39mfLt-AA/edit?disco=AAABKuwDqPg) in the spec, this should be "Made".  

Also, localizations must be done for complete sentences, so the "Code.org" link should be part of the sentence that is localized.  This caters to languages where the word order might change, and/or the translator needs full context to do a proper translation.  We can look at similar app localizations to decide whether it's best done with markdown, inline HTML, or a replaceable token.

Reply:
Updated the string to "Made...".
Re adding link into the locale - agree, though I might use some help here with how to add link into this locale. I've tried to find an example of how we do this in our repo, but didn't find one yet.